### PR TITLE
feat: Add CircularList & Node classes

### PR DESCRIPTION
**Pull request by RogerMo01**
These classes are to handle the play order of players

### CODE REVIEW

File: DominoLibrary/Board.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿namespace DominoLibrary;
+﻿using Utils;
```
Comment (ClauAlvarez01) [inline DominoLibrary/Board.cs:1]: I've never used this class, I didn't even know it existed 😅 Can you explain to me what it is and what made you want to use it? 🤓🙏

Comment (RogerMo01) [reply DominoLibrary/Board.cs:1]: It's a namespace created by me. It's the namespace which contains CircularList & Node classes

Comment (ClauAlvarez01) [reply DominoLibrary/Board.cs:1]: Ok!

**Review by ClauAlvarez01 — APPROVED** [review]
Perfect!

**Pull request merged**